src/card/generate_card_assets.rb

```diff
@@ -8,6 +8,14 @@
 generator = Card::Generators::VipsBasic.new
 storage_adapter = Card::StorageAdapters::GitStorageAdapter.new
 
+card_backs = {
+  grey: "assets/core/card/back/grey.png",
+  blue: "assets/core/card/back/blue.png",
+  green: "assets/core/card/back/green.png",
+  red: "assets/core/card/back/red.png",
+  gold: "assets/core/card/back/gold.png"
+}
```

Comment:
Wasn't sure where best to store this information- I think that eventually we'll want a more flexible tier system and we'll have a better answer to this question at that time.